helm/examples/ldap/README.md

```diff
@@ -0,0 +1,111 @@
+# Containerized HPCC LDAP Support
+
+These examples demonstrate how to enable the HPCC LDAP Security Manager, by adding LDAP administrator account credentials directly to the HPCC helm charts, or by externalizing these credentials using Kubernetes and Hashicorp Vault secrets. Note there are many variations on how to set these values, using yaml, override files and command line settings. To use externalized credentials, you should first run the tutorial on setting up Kubernetes secrets and the Hashicorp Vault, found in the README.md file in the "HPCC-Platform\helm\examples\secrets" folder.
+
+Note that the LDAP Administrator account performs AD directory searches and modifications, and is the only HPCC user that must have LDAP administrator rights.  This account should exist in the configured "systemBasedn" branch of the LDAP AD directory, typically set to cn=Users.
+
+--------------------------------------------------------------------------------------------------------
+
+## Configure LDAP using helm charts and yaml
+
+### Add credentials to ECLWatch ldap.yaml
+   Edit the ECLWatch HPCC-Platform\esp\applications\common\ldap\ldap.yaml (or override file), and fill out the systemCommonName, systemUser, and MD5 encrypted systemPassword entries as follows.
+   Ensure the other settings are correct for your directory, especially the OU settings. Make sure that ldapAdminSecretKey and ldapAdminVaultId are empty.
+
+```bash
+   ldapAdminSecretKey: ""
+   ldapAdminVaultId: ""
+   systemCommonName: "hpcc_admin"
+   systemPassword: "6b3f606902cb9a337904efe2650eb771"
+   systemUser: "hpcc_admin"
+```
+
+### Enable LDAP authentication on the ECLWatch command line
+   Add the following "auth" and "ldapAddress" settings to your ECLWatch command line or override file, ensuring the ldapAddress is the IP of your LDAP server.
+
+```bash
+esp --application=eclwatch --auth=ldap --ldapAddress=10.200.1.2 ...
```

Comment:
This shouldn't be necessary, you should be able to set  "auth: ldap" and "ldapAddress: 10.200.1.2" in values.yaml under the esp instance.  Also needs testing to make sure it's right.